editoast/src/views/catalogue_entry.rs

```diff
@@ -0,0 +1,53 @@
+use database::DbConnectionPoolV2;
+use utoipa::ToSchema;
+
+use crate::error::Result;
+use crate::views::AuthenticationExt;
+use crate::views::AuthorizationError;
+use axum::Extension;
+use axum::extract::Json;
+use axum::extract::State;
+use serde::Deserialize;
+use serde::Serialize;
+use std::sync::Arc;
+
+#[derive(Serialize, Deserialize, ToSchema)]
+pub(in crate::views) struct CatalogueEntryResult {
```

Comment:
why not return the model directly?

Reply:
Is it a good thing to return the model in the views ?

Reply:
Theoretically no, but we still do it in many places for convenience :shrug: